src/myWallet/wallet.lightning.service.js

```
@@ -0,0 +1,424 @@
+                sdk.removeEventListener(listenerId);
+            }
+            if (timeoutId) {
+                clearTimeout(timeoutId);
+            }
+
+            reject(error);
+        }
+    });
+}
+
+export const getLightningTransactions = async (phrase) => {
```

ESLint

[error] Replace `(phrase)` with `phrase`

(prettier/prettier)